src/app/pages/my-request-detail/my-request-detail.component.html

```diff
@@ -59,6 +61,56 @@ <h1>{{ request.label }}</h1>
         <p>{{ request.description }}</p>
       </div>
     </div>
+
+    <div class="mdm-my-request__query">
+      <div class="mdm-my-request__query--row row">
+        <div *ngIf="cohortQuery.rules.length > 0" class="mdm-my-request__query-cohort">
+          <mdm-meql-output [query]="cohortQuery">{{
```

Comment:
Interpolating the `cohortQuery | meql` expression as a child of this component is unecessary, the `mdm-meql-output` component handles all of that when bound to the `query` input.

Reply:
I see, fixed